spongeauth/api/views.py

```diff
@@ -73,6 +74,11 @@ def _create_user(request):
     verified = request.POST.get('verified', 'false') == 'true'
     dummy = request.POST.get('dummy', 'false') == 'true'
 
+    try:
+        accounts.models.validate_username(username)
+    except ValidationError as exc:
+        return django.http.JsonResponse({'errors': exc.messages}, status=422)
```

Comment:
Instead of 422, please use `httplib.UNPROCESSABLE_ENTITY`

(you'll need to `import httplib` too)

Reply:
Whoops, sorry, it's `import http` in Python 3, and `http.HTTPStatus.UNPROCESSABLE_ENTITY`